### ISSUE TITLE
phpcbf.onpaste option

### ISSUE THREAD

**Issue by srikat** [soderlind/vscode-phpcbf#20]
Hi,

It would be nice to have an additional "phpcbf.onpaste" option to automatically format the code when it's pasted into the editor.

**Comment by garrettw** [comment]
Agreed. What it needs to do is provide a formatter that VS Code recognizes, and then I think it should work automatically.

**Comment by garrettw** [comment]
Merging of that PR isn't actually needed because it doesn't really change anything. The recommended config change above will already work.
